recipes/liburing/all/conanfile.py

```diff
@@ -41,12 +48,21 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
-    def requirements(self):
-        self.requires("linux-headers-generic/5.13.9")
+    def get_glibc_version(self):
```

Comment:
```suggestion
    def _get_glibc_version(self):
```

Any method which is not part of Conan API, should be protected. Let's say on next release Conan provides a method with same name, so we have a conflict.